handbook/digital-experience/process-for-submitting-and-publishing-articles.md

```diff
@@ -0,0 +1,98 @@
+# Process for submitting and publishing articles
+
+This guide includes all you need to know about Fleet’s editorial process and how articles are handled between submission and publication.
+
+> For help with formatting your article, check out our [article formatting guide](./article-formatting-guide).
+
+### On this page
+
+- [Article DRIs](#article-dris)
+- [Who can publish articles?](#who-can-publish-articles)
+- [Communication](#communication)
+- [Schedule](#schedule)
+- [How to submit an article](#how-to-submit-an-article)
+- [Review process](#review-process)
+- [Pre-publication checklist](#pre-publication-checklist)
+- [After publication](#after-publication)
+- [Related pages](#related-pages)
+
+## Article DRIs
+
+| Activity | DRI |
+|:---------|:----|
+| Assign & schedule | Tim Kern |
+| Review | Chris McGillicuddy |
+| Artwork | Mike Thomas |
+| Publication | Tim Kern |
+
+## Who can publish articles?
+
+Everyone is welcome to suggest article ideas or submit drafts. To suggest an article, open an issue in the Fleet repo using the article suggestion template (TODO), include a summary, and tag Tim Kern.
+
+## Communication
+
+Use [#help-content-calendar](https://fleetdm.slack.com/archives/C03PH3BBVSM) on Slack for questions and updates related to articles.
+
+## Schedule
+
+Publishing occurs according to our [content calendar](https://docs.google.com/spreadsheets/d/1IzpoIfTc7cqc9g8w20fQfap3GFcrqgOaiPrpCkowcvA/edit?usp=sharing). However, that is subject to change based on the news or information that needs to be posted. For time-sensitive articles, consult with Tim Kern in [#help-content-calendar](https://fleetdm.slack.com/archives/C03PH3BBVSM) on Slack.
+
+## How to submit an article
+
+1. Familiarize yourself with our [article formatting guide](./article-formatting-guide) and writing style guide (todo).
+2. Create a Pull Request for fleetdm.com with a file of your article, formatted in Markdown (todo).
```

Comment:
I don't think "Pull Request" needs to be capitalized. It isn't on GitHub https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests.